ccd-definition/AuthorisationCaseEvent/systemUpdateCUI-nonprod.json

```diff
@@ -4,5 +4,11 @@
     "CaseEventID": "RESET_PIN",
     "UserRole": "caseworker-civil-systemupdate",
     "CRUD": "CRU"
```

Comment:
this should be for release 2 so should be in a new file - systemUpdate-R2-CUI.json

Reply:
Updated as suggested